tlatools/org.lamport.tlatools/src/tla2sany/semantic/SemanticNode.java

```diff
@@ -57,7 +55,16 @@ public SemanticNode(int kind, TreeNode stn) {
     this.tools = EmptyArr;
   }
 
-  public static void setError(Errors errs) { errors = errs; }
+  /**
+   * Set the global static error log for semantic & level checking.
+   *
+   * @param errs The error log instance to use.
+   * @deprecated This method is no longer necessary; all uses of the global
+   * static error log have been removed in favor of passing in the error log
+   * instance at the start of the semantic & level checking processes.
+   */
+  @Deprecated
+  public static void setError(Errors errs) { }
```

Comment:
Why are we keeping `setError`?  Apalache, the only known consumer of SANY, [doesn't seem to call it](https://github.com/search?q=repo%3Aapalache-mc%2Fapalache%20setError&type=code).

Reply:
My experimental interpreter repo does call it, but sure we can just remove it and I'll remove the call from there whenever the snapshot maven package is updated.